doc/ref/configuration/minion.rst

```diff
@@ -635,6 +635,38 @@ FQDN (for instance, Solaris).
 
     append_domain: foo.org
 
+.. conf_minion:: minion_id_remove_domain
+
+``minion_id_remove_domain``
+---------------------
+
+.. versionadded:: 18.3.?
```

Comment:
The version here should be ``Neon``. We will replace with the correct version once we prepare that release.

Reply:
Aah, I first read `None`... 
Set to `Neon`.

I will backport this feature into the msi packaged 18.3.2 because we need it soon.